cmd/buildkitd/config/config.go

```diff
@@ -47,13 +47,22 @@ type OCIConfig struct {
 	Platforms   []string          `toml:"platforms"`
 	Snapshotter string            `toml:"snapshotter"`
 	Rootless    bool              `toml:"rootless"`
+	GCPolicy    []GCPolicy        `toml:"gcpolicy"`
 }
 
 type ContainerdConfig struct {
 	Address   string            `toml:"address"`
 	Enabled   *bool             `toml:"enabled"`
 	Labels    map[string]string `toml:"labels"`
 	Platforms []string          `toml:"platforms"`
+	GCPolicy  []GCPolicy        `toml:"gcpolicy"`
+}
+
+type GCPolicy struct {
+	All          bool     `toml:"all"`
+	KeepBytes    int64    `toml:"keepBytes"`
+	KeepDuration int64    `toml:"keepDuration"`
```

Comment:
time.Duration?

Reply:
It would be cumbersome to use nanoseconds and toml spec doesn't seem to define anything official.